docs/Python-for-Formula-Authors.md

```diff
@@ -40,7 +43,7 @@ If you submit a formula with this syntax to core, you may be asked to rewrite it
 
 ## Applications
 
-`ansible.rb` and `jrnl.rb` are good examples of applications that follow this advice.
+[`ansible.rb`][Formula/ansible.rb] and [`jrnl.rb`][Formula/jrnl.rb] are good examples of applications that follow this advice.
```

Comment:
I used Footnotes link exactly because there are two places mentioning the same formulas.

But if you’d rather remove this section with repetition entirely instead, I’ll do that.